nvflare/app_common/aggregators/dxo_aggregator.py

```diff
@@ -82,8 +92,7 @@ def accept(self, dxo: DXO, contributor_name, contribution_round, fl_ctx: FLConte
 
         processed_algorithm = dxo.get_meta_prop(MetaKey.PROCESSED_ALGORITHM)
         if processed_algorithm is not None:
-            self.log_error(fl_ctx, f"unable to accept DXO processed by {processed_algorithm}")
-            return False
+            self.processed_algorithms.append(processed_algorithm)
```

Comment:
we can do the check here, and instead of a list, we can just keep one item as we are only using one

Reply:
Good point. Changed.